release/_product.sh

```diff
@@ -383,7 +383,7 @@ function start_tomcat {
 
 	for count in {0..30}
 	do
-		if (curl --fail --output /dev/null --silent http://localhost:8080)
+		if (curl --fail --max-time 5 --output /dev/null --silent http://localhost:8080)
```

Comment:
@kiwm, why don't we use 3 seconds and remove the line below (sleep 3) instead?

This way, we wouldn't change the existing logic and also wouldn't change the error message below.

Does it make sense to you?